modules/helm-release/main.tf

```diff
@@ -2,15 +2,31 @@
 # PROVIDER
 # ------------------------------------------------------------------------------
 
+data "external" "client_auth" {
+  program = [
+    "sh", "-c",
+    <<EOF
+      ca_cert=$(cat ${var.client_auth}/ca.cert.pem 2>/dev/null | awk '$1=$1' ORS='  \n')
```

Comment:
FYI I had to look up both the `$1=$1` idiom and `ORS`. Combined with the use of `jq`, it might be worth a comment explaining what all this is doing and why.

If the reason for all of this is to avoid loading a non-existent file, is it possible (and would it be simpler) to create these files elsewhere in Terraform or with rake code instead?

Reply:
Looks like `jq` is capable of handling newlines properly even without `awk` treatment, so that part looks a bit simpler now.

Regarding alternative solutions: well, creating empty cert files at the end of every `destroy`, and then removing them at the beginning of `apply` definitely does not sound better than just fixing provider configuration to not fail when there is no certs!